.github/workflows/deploy-valuesimulator.yaml

```diff
@@ -48,10 +52,16 @@ jobs:
           ssh-private-key: ${{ secrets.DEPLOYMENT_VALUESIM_PRIVKEY }}
       - name: create .ssh/known_hosts
         run: ssh-keyscan -H -t rsa -v vm09.htl-leonding.ac.at  >> ~/.ssh/known_hosts
```

Comment:
maybe abstract the url in a github secret

Reply:
https://vm81.htl-leonding.ac.at/issue/leoiot-122
will be done in the future